docs/about/code-of-conduct.md

```diff
@@ -1,72 +1,132 @@
-# ELIXIR Cloud & AAI Code of Conduct
```

Comment:
I think this should indeed say "ELIXIR Cloud & AAI Code of Conduct". Otherwise we could just link to the original. But rather, we want to reserve the right/opportunity to make our own changes (at the very least adding our own contact point).

The reference to the original is still maintained in the Attribution section.

Reply:
done.